step-templates/octopus-reference-architecture-eks.json

```diff
@@ -0,0 +1,104 @@
\"\n              }\n              \"exec\" = {\n                \"command\" = []\n              }\n              \"failureThreshold\" = \"\"\n              \"periodSeconds\"    = \"\"\n              \"type\"             = \"\"\n              \"httpGet\"          = {\n                \"host\"        = \"\"\n                \"httpHeaders\" = []\n                \"path\"        = \"\"\n                \"port\"        = \"\"\n                \"scheme\"      = \"\"\n              }\n              \"initialDelaySeconds\" = \"\"\n              \"timeoutSeconds\"      = \"\"\n            }\n            \"ReadinessProbe\" = {\n              \"exec\" = {\n                \"command\" = []\n              }\n              \"failureThreshold\" = \"\"\n              \"timeoutSeconds\"   = \"\"\n              \"successThreshold\" = \"\"\n              \"tcpSocket\"        = {\n                \"host\" = \"\"\n                \"port\" = \"\"\n              }\n              \"type\"    = \"\"\n              \"httpGet\" = {\n                \"host\"        = \"\"\n                \"httpHeaders\" = []\n                \"path\"        = \"\"\n                \"port\"        = \"\"\n                \"scheme\"      = \"\"\n              }\n              \"initialDelaySeconds\" = \"\"\n              \"periodSeconds\"       = \"\"\n            }\n            \"TerminationMessagePolicy\"      = \"\"\n            \"VolumeMounts\"                  = []\n            \"ConfigMapEnvFromSource\"        = []\n            \"ConfigMapEnvironmentVariables\" = []\n            \"CreateFeedSecrets\"             = \"False\"\n            \"Lifecycle\"                     = {\n              \"PostStart\" = null\n              \"PreStop\"   = null\n            }\n            \"Name\"         = \"web\"\n            \"StartupProbe\" = {\n              \"successThreshold\" = \"\"\n              \"tcpSocket\"        = {\n                \"host\" = \"\"\n                \"port\" = \"\"\n              }\n              \"failureThreshold\" = \"\"\n              \"httpGet\"          = {\n                \"host\"        = \"\"\n                \"httpHeaders\" = []\n                \"path\"        = \"\"\n                \"port\"        = \"\"\n                \"scheme\"      = \"\"\n              }\n              \"initialDelaySeconds\" = \"\"\n              \"type\"                = \"\"\n              \"exec\"                = {\n                \"command\" = []\n              }\n              \"periodSeconds\"  = \"\"\n              \"timeoutSeconds\" = \"\"\n            }\n          },\n        ])\n        \"Octopus.Action.KubernetesContainers.ServiceName\"         = \"#{Kubernetes.Service.Name}\"\n        \"Octopus.Action.KubernetesContainers.PodManagementPolicy\" = \"OrderedReady\"\n        \"Octopus.Action.Kubernetes.DeploymentTimeout\"             = \"180\"\n        \"Octopus.Action.RunOnServer\"                              = \"true\"\n        \"Octopus.Action.KubernetesContainers.Ingres
```

Comment:
The terraform source code has a few items I want to make sure are expected:

1. Commented out worker pools e.g.
```tf
# Worker Pools
#data "octopusdeploy_worker_pools" "workerpool_linux" {
#  name = "Linux"
#  ids  = null
#  skip = 0
#  take = 1
#}
#
#resource "octopusdeploy_static_worker_pool" "linux" {
#  count       = length(data.octopusdeploy_worker_pools.workerpool_linux.worker_pools) == 0 ? 1 : 0
#  description = "Static workers"
#  is_default  = true
#  name        = "Linux"
#  sort_order  = 3
#}
```
2. Should the sales maven feed be included? e.g.

```tf
resource "octopusdeploy_maven_feed" "feed_sales_maven_feed" {
  count                                = length(data.octopusdeploy_feeds.sales_maven_feed.feeds) == 0 ? 1 : 0
  name                                 = "Sales Maven Feed"
  feed_uri                             = "https://octopus-sales-public-maven-repo.s3.ap-southeast-2.amazonaws.com/snapshot"
  package_acquisition_location_options = ["Server", "ExecutionTarget"]
  download_attempts                    = 3
  download_retry_backoff_seconds       = 20
}
```
3. On Lines 528-672, resources that look like they are to spin up a create a linux worker in a runbook are also commented out e.g. 
```tf
resource "octopusdeploy_runbook_process" "runbook_process_aws_infrastructure_create_linux_worker"`
...
```
5. On Line 925, resource `"octopusdeploy_variable" "frontend_worker_pool"` is commented out
6.

Reply:
Chatted with @mcasperson on this offline